copyright.txt

```diff
@@ -0,0 +1,998 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: Xournal++
+Upstream-Contact: https://github.com/xournalpp/xournalpp/issues/
+Source: https://github.com/xournalpp/xournalpp/
+
+Files: *
+Copyright: 2021, Xournal Team Members
+           2020, Roland Lötscher <roland_loetscher@hotmail.com>
+           2013-2019, Andreas Butti <andreas.butti@gmail.com>
+           2012-2019, Wilson Brenna <wilson@wbrenna.homelinux.com>
+           2015-2019, Marek Pikuła <marek@pikula.tk>
+           2018-2019, Moreno Razzoli
+           2018-2019, Luca Errani <luca.errani1@gmail.com>
+           2019, Ulrich Huber <ulrich@huberulrich.de>
+           2019, Justin Jones <bjustjones@netscape.net>
+           2019, Bryan Tan <Technius@users.noreply.github.com>
+           2019, Justus Rossmeier
+           2019, Fabian Keßler <Fabian_Kessler@gmx.de>
+           2019, lehmanju <internet@devpi.de>
+           2003, Ettore Perazzoli
+           1995, Spencer Kimball and Peter Mattis
```

Comment:
What is our scheme for the years here? Some only have a beginning, others have a period.

Reply:
Those with a beginning only are either not updated or only worked in that year.

Reply:
The way I would understand it, is that:
* `yyyy, Name`: Person still part of active team
* `yyyy-yyyy, Name`: Person no longer part of active team

I think we have two options here:

1. Updating this list by taking the `About` info as source of truth (but being careful with removal of names)
2. Removing all entries and only keep `Xournal Team Members`.

What do you think @Febbe , @rolandlo ?

Reply:
> The way I would understand it, is that:
> 
>     * `yyyy, Name`: Person still part of active team
> 
>     * `yyyy-yyyy, Name`: Person no longer part of active team
> 

What about team members, that have only contributed for 1 year (in the past)? Would that be written like 2016-2016?
 
> I think we have two options here:
> 
>     1. Updating this list by taking the `About` info as source of truth (but being careful with removal of names)
> 
>     2. Removing all entries and only keep `Xournal Team Members`.

The term `Xournal Team Members` does not refer to the Xournal++ team, since Xournal is a different software. There are two files `ZoomCallib.h` and `LoadHandlerHelper.h` that have an `@author Xournal Team` line and refer to  http://xournal.sf.net. That's why `Xournal Team Members` is listed, I think.

> 
> 
> What do you think @Febbe , @rolandlo ?

The `About info` is outdated and it doesn't contain email addresses. A source of truth is https://github.com/xournalpp/xournalpp/graphs/contributors
If "work" means "commits to the master branch, the years/names/email addresses (ordered by the one who was a team member first to the newest team member) would appears to be:

           2012-2018, Wilson Brenna <wilson@wbrenna.homelinux.com>
           2013-2021, Andreas Butti <andreas.butti@gmail.com>
           2015-2016, Marek Pikuła <marek@pikula.tk>
           2017-2019, peetCreative <peetCreative@users.noreply.github.com>
           2018-2019, Moreno Razzoli <morrolinux@users.noreply.github.com>
           2018-2019, Luca Errani <luca.errani1@gmail.com>
           2019-2021, Ulrich Huber <ulrich@huberulrich.de>
           2019-2019, Justin Jones <bjustjones@netscape.net>
           2019-2021, Bryan Tan <Technius@users.noreply.github.com>
           2019-2019, Justus Rossmeier <veecue@users.noreply.github.com>
           2019-2021, Fabian Keßler <Fabian_Kessler@gmx.de>
           2019-2020, Julius Lehmann <internet@devpi.de>
           2020-2021, Tobias Hermann <idotobi@users.noreply.github.com>
           2020-2021, Roland Lötscher <roland_loetscher@hotmail.com>

I don't know the name of peetCreative, but he or she is a team member. There may have been other team members  (maybe @x2b 2013-2015) contributing to Xournal++, but I would not know of that for sure.

           2003, Ettore Perazzoli
           1995, Spencer Kimball and Peter Mattis

are not Xournal++ team members. They have contributed to GTK.

Reply:
> What about team members, that have only contributed for 1 year (in the past)? Would that be written like 2016-2016?

Yes that's weird. We can of course also do it as you proposed. We just need to remember to update the end date (e.g. 2019-2021 will need to be updated in a year). But I guess that should be fine.
Also it's actually more consistent with the other dates.

> The term Xournal Team Members does not refer to the Xournal++ team, since Xournal is a different software.

Ah :facepalm: , I didn't notice the difference. I actually should have seen that in the script, but there was a subtle bug. Good that you pointed me to this otherwise I would have never seen that :pray: .

> I don't know the name of peetCreative, but he or she is a team member.

I added a comment to the `copyright.txt` file that for a complete and full contributor list one should refer to `git log`. I think/hope this should be fine.